epub33/rs/index.html

```diff
@@ -384,14 +384,18 @@ <h3>Network access</h3>
 			<section id="sec-epub-rs-external-links">
 				<h3>External links</h3>
 
-				<p data-tests="#pub-external-links">Reading systems SHOULD open links that resolve outside the <a>EPUB
-						publication</a> in a new browser instance to ensure that the browser's security and privacy
-					controls are available to users.</p>
+				<p><span>Reading systems SHOULD alert users before opening a link that declares a <a
+							data-cite="url#concept-url-scheme">scheme</a> [[url]] (i.e., that resolves to web content
```

Comment:
This seems vague, and I can't say I have done an exhaustive review to say if this is entirely correct. For instance, data:, blob:, and chrome: are all url schemes that are local or even part of the publication. I am not sure if you can "open a link" to data: or blob:, but I am also not sure that you can't.